tests/cases/integration/Relationships/relationships.oneHasMany.phpt

```diff
@@ -342,12 +353,12 @@ class RelationshipOneHasManyTest extends DataTestCase
 	{
 		$books = $this->orm->books->findBy([
 			ICollection::OR,
-			['tags->id' => [1]],
-			['tags->id' => null], // no match
+			['tags->id' => [1]], // #1
+			['tags->id' => null], // matches a book without tags (#4)
 		]);
 
-		Assert::same(1, $books->countStored());
-		Assert::same(1, $books->count());
+		Assert::same(2, $books->countStored());
+		Assert::same(2, $books->count());
```

Comment:
The changed behavior here actually matches Dbal behavior in Nextras 4. The original behavior matched Array behavior in Nextras 4 (i.e. the behavior was inconsistent). Are we ok with this changed behavior? 🤔 Most probably yes. cc @stepapo

Reply:
Surely this is ok, makes things easier. Before this, `[CompareEqualsFunction::class, [CountAggregateFunction::class, 'tags->id'], 0]` had to be used for matching book with no tags.